packages/field-content/src/views/editor/index.js

```diff
@@ -44,12 +44,12 @@ function Stories({ value: editorState, onChange, blocks, className }) {
           if (typeof block.getPlugins !== 'function') {
             return combinedPlugins;
           }
-          return [...combinedPlugins, block.getPlugins({ blocks })];
+          return [...combinedPlugins, ...block.getPlugins({ blocks })];
         },
         [
           ...markPlugins,
           {
-            renderNode(props) {
+            renderBlock(props) {
```

Comment:
I'm pretty sure this will break with links since they're not Slate blocks, they're inlines.

Reply:
Excellent catch! I've had to add a `renderInline` as well as a `renderBlock` to get the original behaviour back. All tested and working now 👍